### DISCUSSION TITLE
Problem Validating Contract

### DISCUSSION THREAD

**Discussion by RecoX** [hashlips-lab/nft-erc721-collection#439] [Q&A] [answered]
I have a problem when I try to validate the contract

yarn verify 0xbac5036c92ad2ae8feac87f61a8dd999735f4765 —network truffle
yarn run v1.22.15
$ hardhat verify --constructor-args config/ContractArguments.ts 0xbac5036c92ad2ae8feac87f61a8dd999735f4765 —network truffle
Error in plugin @nomiclabs/hardhat-etherscan: The selected network is hardhat. Please select a network supported by Etherscan.

For more info run Hardhat with --show-stack-traces

Any help please?

**Comment by RecoX** [comment, marked as answer]
Its a typo error, here the command working

yarn verify 0xbac5036c92ad2ae8feac87f61a8dd999735f4765 --network truffle

**Reply by liarco** [reply]
Hi @RecoX, I don't see any typo there, it looks like you had your MetaMask wallet connected to the local network instead of the mainnet while you run the command.

I hope this helps

**Reply by RecoX** [reply]
> Hi @RecoX, I don't see any typo there, it looks like you had your MetaMask wallet connected to the local network instead of the mainnet while you run the command.
> 
> I hope this helps

Thanks for your answer. The typo is because I used a sticky notes to write all the commands in your videos.


For some reason it converts the character `-` to `—` hence the weird error.

Everything works fine, excellent the new way of using truffle as a proxy for the CLI commands, thanks for your tutorial and code

[Watermelon In Easter Hay]

Best.

**Reply by liarco** [reply]
Hi @RecoX, you are right, now I see it, thanks for pointing that out. That's definitely useful info!

I'm happy you like it, I wish you the best of luck with your projects.
